[docs] Include File Transfers in moderated sessions docs (#26032)
gravitational/teleport: docs/pages/access-controls/guides/moderated-sessions.mdx

This information can be picked up by a third party integration and may for
example be used to enable notifications over some external communication system.

## File Transfers

File transfers within moderated sessions are available via the Web UI only. When initiating a file transfer, if the
current active session requires moderation, a file transfer request will automatically be sent to all current party members.

Both the session originator and the moderator(s) must be present in the Web UI during the file transfer initiation to receive
the file transfer request notification. Once the file transfer has been requested, all members of the party will be notified
and prompted with an Approve/Deny dialog.

> If a Deny is sent, the file transfer request is immediately removed and all party members are notified.

Reviewer: ```suggestion
If a moderator denies the file transfer request is immediately removed and all party members are notified.
```

After enough approvals have been given to satisfy the policy (the same policy to start the session), the file transfer
will automatically begin.

> 

Reviewer 2: It might also make sense to have an FAQ or troubleshooting page with what happens when you try to `tsh scp` when moderation is required.

Author: Good idea, will add!

Reviewer: having some screen shots would be an added bonus.

Reviewer: Are file approve/denies logged as audit log entries?  good to note if they are.

Author: Not yet, have an upcoming PR that will add the audit logs and will update the docs then. The file transfer itself is still audited as usual

This information can be picked up by a third party integration and may for
example be used to enable notifications over some external communication system.

> ## File Transfers

Reviewer 3: ```suggestion
## File transfers
```
Minor style suggestion: we use sentence case for headers

File transfers within moderated sessions are available via the Web UI only. When initiating a file transfer, if the
current active session requires moderation, a file transfer request will automatically be sent to all current party members.

Both the session originator and the moderator(s) must be present in the Web UI during the file transfer initiation to receive
the file transfer request notification. Once the file transfer has been requested, all members of the party will be notified
and prompted with an Approve/Deny dialog.

![Approve/Deny Prompt](https://github.com/gravitational/teleport/assets/5201977/49e11b8c-4729-42b4-a83c-9db7dfa25b74)

> If a moderator denies the file transfer request is immediately removed and all party members are notified.

Reviewer 3: ```suggestion
If a moderator denies the file transfer request, the file is immediately removed and all party members are notified.
```
Or "the request"?

the file transfer request notification. Once the file transfer has been requested, all members of the party will be notified
and prompted with an Approve/Deny dialog.

> ![Screenshot 2023-05-12 at 1 28 35 PM](https://github.com/gravitational/teleport/assets/5201977/49e11b8c-4729-42b4-a83c-9db7dfa25b74)

Reviewer: You are supposed to add this as a image and describe it. 
```suggestion
![Approve/Deny Prompt](https://github.com/gravitational/teleport/assets/5201977/49e11b8c-4729-42b4-a83c-9db7dfa25b74)
```
